src/components/Create/BudgetForm.tsx

```diff
@@ -46,22 +46,38 @@ export default function BudgetForm({
 
   return (
     <Space direction="vertical" size="large" style={{ width: '100%' }}>
-      <h1>Funding</h1>
+      <h1>Setup Your Funding Cycle</h1>
 
       <Form layout="vertical">
         <div style={{ color: colors.text.secondary }}>
-          <h4>Target</h4>
+          <h4>Funding Cycle</h4>
           <p>
-            No more than the target can be distributed from the project in a
-            single funding cycle. Whenever a new funding cycle starts, any
-            overflow automatically goes towards that cycle's target amount,
-            acting as a project's runway.
+            A funding cycle is how your project will manage its funds. A funding
```

Comment:
It might be useful mentioning that funding cycles can evolve over time on a per-funding cycle basis. You can start with a simple configuration and add parameters over time as a project matures.

Setting "no duration" gives the project total flexibility to reconfigure the funding cycle on demand. With a duration in place, a configuration is locked in and can only be changed for subsequent cycles.